static/src/stylesheets/module/content/_article-minute.scss

```diff
@@ -571,8 +571,12 @@ $minute-layout-margin-nudge: gs-span(1);
                     width: 100%;
                 }
 
-                .gu-image {
+                .element--inline__outer {
```

Comment:
hmm, the parent element has the `element--inline` class and `outer` is a descendent. 
that made sense to me, but i'm not a BEM master :(